<?xml version="1.0" ?>
<launch xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="https://gist.githubusercontent.com/nalt/dfa2abc9d2e3ae4feb82ca5608090387/raw/roslaunch.xsd">

        
  <group ns="niryo_robot_hardware_interface" >
      <group ns="can_driver">
        <rosparam file="$(find can_driver)/config/can_bus.yaml" command="load"/>
        <rosparam file="$(find can_driver)/config/motors_config.yaml" command="load"/>
        <rosparam file="$(find can_driver)/config/default.yaml" command="load"/>  
      </group>

      <node pkg="conveyor_interface" type="conveyor_interface_node" name="conveyor_interface" output="screen">  
        <rosparam file="$(find conveyor_interface)/config/default.yaml" command="load"/>
      </node>
  </group>
  
</launch>
